### PR TITLE
Add mobile application page

### LINKED ISSUE #16: Mobile Applications
- Content

### PR DESCRIPTION
**Pull request by Afrasyabafzal**

### CODE REVIEW

File: mobile-application.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (hamzafer) [inline mobile-application.html:4]: Add all the other tags for example: 'google analytics', 'tag manager' tags as well.

See other HTML files for reference.

**Review by hamzafer — CHANGES_REQUESTED** [review]

File: mobile-application.html (hunk 1 above)
Comment (Afrasyabafzal) [reply mobile-application.html:4]: Done

File: mobile-application.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (hamzafer) [inline mobile-application.html]: what is this tag for?

Comment (hamzafer) [reply mobile-application.html]: content="IE=edge"

File: mobile-application.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="google-site-verification" content="QyzRhR9f4u41ua5XJksjItnBUUquJvU_gCz8Hz3z2fc" />
+    <title>IOS Application and Android Section </title>
```
Comment (hamzafer) [inline mobile-application.html]: remove extra space

Comment (hamzafer) [reply mobile-application.html]: at the end

File: mobile-application.html [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+          </section>
+        </main>
+      </div>
+      <div id="footer"></div>
+      <script type="text/javascript" id="hs-script-loader" async defer src="//js-na1.hs-scripts.com/21922964.js"></script>
```
Comment (hamzafer) [inline mobile-application.html]: what is this script?

**Review by hamzafer — CHANGES_REQUESTED** [review]

File: mobile-application.html (hunk 2 above)
Comment (Afrasyabafzal) [reply mobile-application.html]: this is auto generated by vs code

File: mobile-application.html (hunk 4 above)
Comment (Afrasyabafzal) [reply mobile-application.html]: The chat bot script

File: mobile-application.html (hunk 3 above)
Comment (Afrasyabafzal) [reply mobile-application.html]: Done

**Review by hamzafer — APPROVED** [review]

**Pull request merged**